docs/installation/index.rst

```diff
@@ -0,0 +1,46 @@
+.. _installation:
+
+Installation
+============
+
+mlir-graphblas is implemented via Python and Cython.
+
+Building from source is possible, but the recommended method to install is via ``conda``.
+
+It is assumed by mlir-graphblas that it is installed using ``conda`` or ``venv``. Thus, installing using one of those two methods is recommended.
+
+Python version support
+----------------------
+
+Python 3.8 and above is supported.
+
+Installing using conda
+----------------------
+
+::
+
+    conda install -c conda-forge -c metagraph mlir-graphblas
+
+Installing from source
+----------------------
+
+Instructions for building from source are currently a work in progress.
+
+Installing using venv
+---------------------
+
+Instructions for installing via venv are currently a work in progress.
+
+Required Dependencies
+---------------------
+
+These should be automatically installed when mlir-graphblas is installed
+
+  - `numpy <https://numpy.org>`__
+  - `PyMLIR <https://github.com/metagraph-dev/pymlir>`__
+  - `lvmlite <https://llvmlite.readthedocs.io/en/latest/>`__
```

Comment:
Typo in the name

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/26/commits/46b00a45f1152e319f2d3fd7d10134e6ee16172a fixes the typo.